application/src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -56,14 +60,31 @@ public DirectTransferGenerator(
     Graph graph,
```

Comment:
Is this constructor used only for testing? If so, can you add Javadoc?

Reply:
I added a short javadoc